pkg/util/ec2/ec2.go

```diff
@@ -144,12 +145,15 @@ func extractClusterName(tags []string) (string, error) {
 	return clusterName, nil
 }
 
-func getResponse(url string) (*http.Response, error) {
+func getResponse(url string, method string, headers map[string]string) (*http.Response, error) {
```

Comment:
What do you think about updating the name of the function as it can perform put request now?

Reply:
I'm open to suggestion, I though about `doRequest` not sure it is really better that `getResponse` (in some extend we still get a <method> response in all situation 🤔)

Reply:
I will go with doHTTPRequest(...)